libultraship/libultraship/SDLController.h

```diff
@@ -5,33 +5,43 @@
 namespace Ship {
 	class SDLController : public Controller {
 		public:
-			SDLController(int slot) : Controller(), Cont(nullptr), physicalSlot(slot) { }
-			void ReadFromSource(int32_t slot) override;
-			const char* GetControllerName() override;
-			const char* GetButtonName(int slot, int n64Button) override;
-			void WriteToSource(int32_t slot, ControllerCallback* controller) override;
+			inline static const char* AxisNames[] = {
+				"Left Stick X",
+				"Left Stick Y",
+				"Right Stick X",
+				"Right Stick Y",
+				"Left Trigger",
+				"Right Trigger",
+				"Start Button"
```

Comment:
start button is an axis name? i'm assuming this existed before but it seems odd

Reply:
This existed before. Not sure why.